fields/types/html/editor/mixins/entity-editing-block-mixin.js

```diff
@@ -230,8 +232,7 @@ let EntityEditingBlock = (superclass) => class extends Component {
 
 // block settings
 const BLOCK_TYPES = [
-	{ label: 'H3', style: 'header-three', icon: 'fa-header', text: '3' },
-	{ label: 'H4', style: 'header-four', icon: 'fa-header', text: '4' },
+	{ label: 'H2', style: 'header-three', icon: 'fa-header', text: '2' },
```

Comment:
I just found in 'HtmlField.js', the style is 'header-two'. maybe it could be replaced as 'header-two' to maintain the consistency (?